lms/djangoapps/instructor/management/commands/collect_course_forums_data.py

```diff
@@ -0,0 +1,51 @@
+"""
+Command to retrieve aggregated course forums data in a .csv
+"""
+import csv
+import optparse
+import os
+
+from instructor.utils import collect_course_forums_data
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from instructor.management.commands.collect_ora2_data import preprocess
+
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    """
+    Query aggregated open assessment data, write to .csv
+    """
+
+    help = ("Usage: collect_course_forums_data <course_id> --output-dir=<output_dir>")
+    args = "<course_id>"
+    option_list = BaseCommand.option_list + (
+        optparse.make_option('-o', '--output-dir',
+                             action='store', dest='output_dir', default=None,
+                             help="Write output to a directory rather than stdout"),
+    )
+
+    def handle(self, *args, **options):
+        if not args:
+            raise CommandError("Course ID must be specified to fetch data")
+
+        try:
+            course_id = SlashSeparatedCourseKey.from_deprecated_string(args[0].decode('utf-8'))
+        except InvalidKeyError:
+            raise CommandError("The course ID given was invalid")
+
+        file_name = ("%s-forums.csv" % course_id).replace("/", "-")
+
+        if options['output_dir']:
+            csv_file = open(os.path.join(options['output_dir'], file_name), 'wb')
+        else:
+            csv_file = self.stdout
+
+        writer = csv.writer(csv_file, dialect='excel', quotechar='"', quoting=csv.QUOTE_ALL)
+
+        header, rows = collect_course_forums_data(course_id)
+
+        writer.writerow(header)
+        for row in rows:
+            writer.writerow(preprocess(row))
```

Comment:
I'd rather not have to cherrypick dylan's PR just for this preprocess function. You could just define a function right above this like:
```
def _utf8_encoded_row(row):
    return [unicode(item).encode('utf-8') for item in row]
```